foundry.md

```diff
@@ -198,8 +198,10 @@ The values are forwarded to the `#setCode` marker which updates the account acco
 
 ```k
     rule [call.etch]:
-         <k> CALL _ CHEAT_ADDR _VALUE ARGSTART ARGWIDTH _RETSTART _RETWIDTH
-          => #setCode(#asWord(#range(LM, ARGSTART +Int 4, 32)), #range(LM, ARGSTART +Int 36, ARGWIDTH -Int 36))
+         <k> CALL _ CHEAT_ADDR _VALUE ARGSTART _ARGWIDTH _RETSTART _RETWIDTH
+          => #let CodeStart = ARGSTART +Int 100 #in
+             #let CodeLen = #asWord(#range(LM, ARGSTART +Int 68, 32)) #in
+             #setCode(#asWord(#range(LM, ARGSTART +Int 4, 32)), #range(LM, CodeStart, CodeLen))
           ~> 1 ~> #push
```

Comment:
all-caps logical variable names (`CodeStart` and `CodeLen`). This would be the only place in the code where not all-caps variables are used.

Reply:
Good point, fixed!